client/devdocs/example.js

```diff
@@ -55,17 +55,17 @@ class Example extends Component {
 			Gridicon,
 		};
 
-		return code ? (
-			<LiveProvider
-				code={ code }
-				scope={ scope }
-				className="woocommerce-devdocs__example"
-				noInline={ true }
-			>
-				<LiveError />
-				<LivePreview />
-			</LiveProvider>
-		) : null;
+		return code ? <th code={ code } scope={ scope } /> : null;
```

Comment:
Let's just return null for now, the `th` is throwing warnings about semantics & wouldn't display anything here anyway.